Resources/config/payum.xml

```diff
@@ -49,7 +49,9 @@
         <service id="payum" class="Payum\Core\Payum">
             <factory service="payum.builder" method="getPayum" />
         </service>
-        <service id="Payum\Core\Payum" alias="payum"></service>
+        <service id="Payum\Core\Payum" alias="payum"/>
+        <!-- Allows to bind Payum as RegistryInterface also -->
+        <service id="Payum\Core\Registry\RegistryInterface" alias="Payum\Core\Payum"/>
```

Comment:
I don't think this is good idea. I'm not totally sure, but it may cause issues. At least it should be in it's own PR.

Reply:
If we follow the idea of below comment, then this is not needed. I will remove it